src/components/multi_select.js

```diff
@@ -100,7 +102,9 @@ export class MultiSelect extends PureComponent {
       maxSelectedItems,
       searchValue,
       withGrouping,
-      generateClassName
+      generateClassName,
+      listRenderer,
+      selectedItemHeight
```

Comment:
This was the "additional feature" :)

Could you also make sure that the change is reflected in the Readme file to ensure that users know how to use it properly?
Mainly we should expose the interface expected of listRenderer to work.

Reply:
Okay. I can separate this for 2 PRs. `selectedItemHeight` also needed to implement that Multiple Image select.

Reply:
@liorheber resolved your comments. I will create new PR for the `selectedItemHeight`